lib/wovnrb/version.rb

```diff
@@ -1,3 +1,3 @@
 module Wovnrb
-  VERSION = '3.11.1'.freeze
+  VERSION = '4.1.0'.freeze
```

Comment:
I don't think this is intended? Perhaps:
```suggestion
  VERSION = '3.12.0'.freeze
```